demos-europe/demosplan-core
# Use local pdflatex api & docx import container instead of java service

A step towards smaller containers away from fat containers is to use several containers for distinct services. This PR introduces and activates two new containers for docx import and pdf export.

### How to review/test
This is the last PR of a series of PRs that needs to be merged first. When they are you should restart your containers and docx import and pdf export should just work. You can verify that the new containers are used when you see a request in the symfony toolbar for HTTP Client

### Linked PRs (optional)


### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
